.github/workflows/anaconda_debian_ci.yml

```diff
@@ -18,15 +20,25 @@ jobs:
 
       - name: Login to DockerHub
         uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
-        if: github.ref == 'refs/heads/master'
+        if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags')
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to Public ECR
```

Comment:
Does public.ecr.aws/y0o4y9o3/anaconda3 exist? Afaik we only push the anaconda-pkg-build images to ECR to have rate limits during package builds.